react/components/ProductAssemblyOptions/ProductAssemblyOptionItemCustomize.tsx

```diff
@@ -90,13 +90,28 @@ const ModalView: FC<{ closeAction: () => void }> = ({
 
 interface Props {
   buttonProps?: ButtonProps
-  modalProps?: unknown
+  modalProps?: any
 }
 
 interface ButtonProps {
   collapse?: 'left' | 'right' | 'none'
 }
 
+const hasValues = (itemChildren: any) => {
+  const [name] = Object.getOwnPropertyNames(itemChildren)
```

Comment:
Why do you only check for the first item?

Reply:
Because it's always the name